opal/corelib/enumerable.rb

```diff
@@ -177,6 +177,10 @@ def count(object = undefined, &block)
   end
 
   def cycle(n = nil, &block)
+    if `arguments.length > 1`
```

Comment:
You have to explicitly enable arity checking, but I'm unsure how it's done with `opal-repl` or if it's doable at all.

@elia @wied03 is it possible something changed in the rake tasks that disabled arity checking? Or is this case not accounted by it?